test/rubygems/test_gem_specification.rb

```diff
@@ -1210,14 +1206,6 @@ def test_self_stubs_for_mult_platforms
 
   DATA_PATH = File.expand_path "../data", __FILE__
 
-  def test_handles_private_null_type
-    path = File.join DATA_PATH, "null-type.gemspec.rz"
-
-    data = Marshal.load Gem::Util.inflate(Gem.read_binary(path))
-
-    assert_nil data.rubyforge_project
```

Comment:
Can't we just change assertion in here? What's purpose of this test?

Reply:
I went through the history (blame) on this file and the latest commits on each line weren't too illuminating: https://github.com/rubygems/rubygems/blame/master/test/rubygems/test_gem_specification.rb

However nothing about this test makes me think that `rubyforge_project` is the necessary component. We can flag up some other authors but I could preserve this test with the assertion looking at a different attribute. (say 'description'). Will commit that now.

Reply:
Yup, that's what I was thinking about as well. It is not clear what's the purpose of this test at all, but it is probably not directly testing `rubyforge_project`, but uses `rubyforge_project` to assert only. Changing assertion to use different attribute should work here also.

Reply:
Interestingly enough, as I just found out, `description` does not assert nil. I am wondering if this is only actually passing on an `assert_nil` because `rubyforge_project` simply isn't set anymore.....